src/main/java/edu/cornell/kfs/sys/dataaccess/DtoPropertyHandler.java

```diff
@@ -0,0 +1,47 @@
+package edu.cornell.kfs.sys.dataaccess;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.function.BiConsumer;
+
+import org.apache.commons.lang3.function.FailableBiFunction;
+
+public final class DtoPropertyHandler<T, V> {
```

Comment:
I commented earlier about what this new class buys us

Reply:
Will update or remove this class accordingly as part of the refactoring work.